dnf-behave-tests/features/unprivileged-installroot-2.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Installroot test as unprivileged user
+
+
+@bz1843280
+@destructive
+@no_installroot
+Scenario: Fail when missing permissions for installroot directory
+  Given I use repository "miscellaneous"
+   When I execute dnf with args "--releasever=32 --installroot=/var/lib/f32 install dummy" as an unprivileged user
+   Then the exit code is 1
+    And stderr is
+        """
+        Config error: [Errno 13] Permission denied: '/var/lib/f32': '/var/lib/f32'
+        """
```

Comment:
AFAIK you can now put the two scenarios into a single .feature file and the runner will run them in separate containers if they are `@no_installroot` (you can also mark the whole `Feature` `@no_installroot` I think).

Reply:
Hm, I guess I must have run it without the runner, because I thought for sure it was failing. All right then, I put it into one file.